reconcile/utils/terrascript_aws_client.py

```diff
@@ -1409,10 +1416,26 @@ def populate_tf_resource_rds(self, spec):
         # this will only affect the output Secret
         output_resource_db_name = values.pop("output_resource_db_name", None)
 
+        if "event_notifications" in values:
+            event_notifications = values.pop("event_notifications")
+            for e_n in event_notifications:
+                sns_topic_name = e_n.pop("destination")
+                if sns_topic_name.startswith("arn:"):
+                    raise (ValueError("destination should not be an arn"))
```

Comment:
Will this error be thrown during dry-run mode so we can catch it during PR check?

Reply:
Correct! This will be thrown when --dry-run is enabled.